From 7e3ba421018f79d8cc8d3ea1c379087f2cadd388 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Mon, 20 Jun 2022 06:52:33 +0300
Subject: [PATCH] [SW-89971] rom: remove redundant rr config

We can remove programming rr idx 0 as it is redundant. Earlier
idx 0 was used as a backup rr config in ROM in case of HW rr config
failure, but as this logic was removed, programming of idx 0 now is
redundant.

Change-Id: Idf6e57e2f6b8e9a0ea77661a2e1b1de1da4381e6
---

diff --git a/arch/gaudi3/arc/security.c b/arch/gaudi3/arc/security.c
index 323e02b..e42944a 100644
--- a/arch/gaudi3/arc/security.c
+++ b/arch/gaudi3/arc/security.c
@@ -745,14 +745,8 @@
 {
 	security_handle_twi();
 
-	rr_ctx_lbw[RR_OPEN_ALL].rr_idx = HL_SEC_LBW_RR_SHORT_0;
-	security_rr_config_lbw_if_common(&rr_ctx_lbw[RR_OPEN_ALL]);
-
 	rr_ctx_lbw[RR_OPEN_ALL].rr_idx = HL_SEC_LBW_RR_SHORT_13;
 	security_rr_config_lbw_if_common(&rr_ctx_lbw[RR_OPEN_ALL]);
-
-	rr_ctx_hbw[RR_OPEN_ALL].rr_idx = HL_SEC_HBW_RR_SHORT_0;
-	security_rr_config_hbw_if_common(&rr_ctx_hbw[RR_OPEN_ALL]);
 
 	rr_ctx_hbw[RR_OPEN_ALL].rr_idx = HL_SEC_HBW_RR_SHORT_7;
 	security_rr_config_hbw_if_common(&rr_ctx_hbw[RR_OPEN_ALL]);
